#A vector of bounding box messages

BoundingBox[] bounding_box_vector